src/routes/registration/+page.svelte

```diff
@@ -25,20 +26,37 @@
 		<InfoBox title="The JSON format">
 			<p>Your registration as JSON file should be in the following format to pass the tests:</p>
 			<pre>{`{
-    "name": "Your real name (OR a nickname)",
-    "company": "your company name (optional)",
+    "name": "your name",
+    "company": "your company name",
     "when": {
+        // 2023-06-30
         "friday": true/false,
+        // 2023-07-01
         "saturday": true/false
     },
-    "tags": ["any", "tag", "you like"],
-    "vegan": true/false,
-    "vegetarian": true/false,
-    "allergies": "... (optional)",
-    "what_is_my_connection_to_javascript": "Say something about you and JS ...",
-    "what_can_i_contribute": "How can you contribute to the event? If you don't know yet, feel free to give it some thought ...",
-    "tshirt": "W-S",
-    "twitter": "handle (optional)"
+    // if you are willing to take session notes and publish them to github (required)
+    "iCanTakeNotesDuringSessions": true/false,
+    // your current interests (JS and in general) (required)
+    "tags": ["ReScript", "Svelte", "Zustand", "GraphQL"],
+    // if you only eat vegan food (optional)
+    "vegan": false,
+    // if you only eat vegan or vegetarian food (optional)
+    "vegetarian": true,
+    // what you cannot eat or drink (optional)
+    "allergies": ["Gluten", "Peanuts"],
+    // tell us a few words how JavaScript affects you (required)
+    "whatIsMyConnectionToJavascript": "...",
+    // what can you contribute to the bar camp (required)
+    "whatCanIContribute": "???",
+    // if you want a T-Shirt we need your size and variant preference (optional)
+    "tShirt": {
+        // S | M | L | XL | XXL
+        "size": "S",
+        // fitted (also known as waist cut or women variant) or regular
+        "type": "fitted"
+    },
+    "twitter": "yourhandle",
+    "mastodon": "your Mastodon URL"
```

Comment:
Could we add the comments to make it clear that they are optional, like others, above?

How about:
```suggestion
    // optional
    "website": "URL - your home page or website",
    // optional
    "twitter": "yourhandle",
    // optional 
    "mastodon": "your Mastodon URL"
```